### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by coker2019**
Created a Mobile Menu and made it dynamic with Javascript.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  event.preventDefault();
+});
+
+closed.addEventListener('click', () => {
+  mobileMenu.style.display = 'none';
+});
+
+// mobileList.addEventListener('click', () =>{
+//   mobileMenu.style.display = 'none';
+// });
```
Comment (omar-muhamad) [inline index.js]: - [ ] Kindly remove any commented code to make your code clean from any unused code.

File: style.css [hunk 2]
```diff
@@ -42,6 +42,45 @@ header {
...
+
+.mobile-list {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  margin: auto;
+  padding: 20px;
+  width: 80%;
+  font-size: 2rem;
+}
```
Comment (omar-muhamad) [inline style.css]: Even though the mobile menu functionality is working as expected but you need to change the following:

- [ ] Kindly make the mobile menu background take the whole width of the page, so kindly remove the white margin at the top.
- [ ] Please add the correct font family and the correct placement for the mobile menu options by moving them to the left to match the given design.


| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi Team,

### Nice job, you are almost there.

cat-driving-serious

## (Highlights) Good Points: :+1:
- Your work is well documented. ✔
- The correct workflow is used.  ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below ⬇️.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.js (hunk 1 above)
Comment (LionRouge1) [reply index.js]: [OPTIONAL]
- [ ] As requested by the previous review, kindly remove commented code from your code base it make your code unclean

**Review by LionRouge1 — APPROVED** [review]
Hi @coker2019, 

# STATUS: APPROVED! 🥇 💯 
Great work on making the changes 👏 
You've done well implementing the requested changes

done

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by zieeco — COMMENTED** [review]
# STATUS: INVALID!!!  :x: :x: :x: 

Hi @coker2019,




While you made a great effort in this project, unfortunately, I cannot proceed to review your code, please make sure you implement the following changes before you resubmit for review. 

## Required Changes ♻️
_Check the comments below._


- [ ] As it is a requirement for this project, kindly ensure you've worked with your newly paired partner as I see you're repeating the week and you've submitted a PR that you worked with someone else as in the past. This is a new week, so automatically,  you should be paired with another team member. kindly collaborate with them and submit a newly created PR.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Invalid Code Review Request **Does not count** into the code reviews limit._
# **INVALID PROJECT**

**Pull request merged**